Flabbergasted-Flamingos/Bangazon-admin-app
# Cd department detail

# Description
Added department detail view.

## Number of Fixes
Type of number of fixes included in this PR.

## Related Ticket(s)
#7 
## Problem to Solve
department detail view

## Proposed Changes


## Expected Behavior
Clicking on a department in the department list view should take you to a detail view that includes each employee in that department with a link to that employees detail view.

## Steps to Test Solution

```
git fetch --all
git checkout cd_department_detail
python manage.py runserver
```
1. Go to the department list page.
2. Click a department name, and it should take you to the detail view.
3. Click an employee link, and it should take you to the employee detail view.

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
